convert2rhel/utils.py

```diff
@@ -57,6 +58,14 @@ def require_root():
         loggerinst.critical("The tool needs to be run under the root user.")
     return
 
+def safe_repr(obj, short=False):
```

Comment:
Please add a docstring about what's the purpose of the function.
Also, because it's just used for unit tests, I'd move it to https://github.com/oamg/convert2rhel/blob/master/convert2rhel/unit_tests/__init__.py.